base/intfuncs.jl

```diff
@@ -198,7 +198,7 @@ end
 function ndigits0z(n::Unsigned, b::Int)
     d = 0
     if b < 0
-        d = ndigits0znb(signed(n), b)
+        d = ndigits0znb(n, b)
     else
```

Comment:
CI has been hanging all along because of this (see https://github.com/JuliaLang/julia/blob/master/test/intfuncs.jl#L78)